lib/metadata/get-assignee.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var github = require('../github');
+
+module.exports = function getAssignee(number, metadata) {
+    return github.get('/repos/:owner/:repo/issues/:number', { number: number }).then(function(issue) {
+      if (issue.assignee) {
+        return null;
+      }
+      var reviewers = metadata.reviewersExcludingAuthor.map(function (owner) {
+        return owner.login
```

Comment:
This statement is missing a semicolon, but the map seems superfluous. How about grabbing the `login` property in the final `return` statement instead?

Reply:
Done.